spec/helpers/work_packages_helper_spec.rb

```diff
@@ -32,7 +32,12 @@
   let(:stub_work_package) { FactoryGirl.build_stubbed(:work_package) }
   let(:stub_project) { FactoryGirl.build_stubbed(:project) }
   let(:stub_type) { FactoryGirl.build_stubbed(:type) }
-  let(:form) { double('form', select: '').as_null_object }
+  let(:label_placeholder) { "<label>blubs</label>".html_safe }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.